### PR TITLE
Update the README to link out to current tutorial

### PR DESCRIPTION
**Pull request by avdv**
# Pull Request Checklist

* [x] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
* [x] Have you read through the [contributor guidelines](https://github.com/ReactiveMongo/ReactiveMongo/blob/master/CONTRIBUTING.md#reactivemongo-developer--contributor-guidelines)?
* [x] Have you [squashed your commits](https://www.playframework.com/documentation/2.4.x/WorkingWithGit#Squashing-commits)?
* [x] Have you added tests for any changed functionality?

### CODE REVIEW

**Comment by mkurz** [comment]
@cchantep You did not push the 0.16.2 git tag to the GitHub repo...

File: README.md [hunk 1]
```diff
@@ -1,22 +1,25 @@
 # ReactiveMongo for Play Framework
 
-This is a plugin for [Play Framework](https://www.playframework.com) 2.4 and 2.5, enabling support for [ReactiveMongo](http://reactivemongo.org) – a reactive, asynchronous and non-blocking Scala driver for MongoDB.
+This is a module for the [Play Framework](https://www.playframework.com) 2.5, 2.6 and 2.7, enabling support for [ReactiveMongo](http://reactivemongo.org) – a reactive, asynchronous and non-blocking Scala driver for MongoDB.
 
 ## Usage
 
 In your `project/Build.scala` or `build.sbt`:
```
Comment (cchantep) [inline README.md]: I would rather remove the section content and add a link to the documentation site, to avoid duplication.

Comment (avdv) [reply README.md]: OK, sure. This link? http://reactivemongo.org/releases/0.1x/documentation/tutorial/play.html

Comment (cchantep) [reply README.md]: yes

**Pull request merged**